connect/resources/directory.py

```diff
@@ -53,7 +53,7 @@ def list_marketplaces(self, filters=None):
 
         :param dict|Query filters: Filters to pass to the request.
         :return: A list with the marketplaces that match the given filters.
```

Comment:
List of marketplaces matching given filters

Reply:
Fixed.